modules/openapi-generator/src/main/resources/python/api.mustache

```diff
@@ -7,9 +7,6 @@ import io
 import warnings
 
 from pydantic import validate_arguments, ValidationError
-{{#asyncio}}
-from typing import overload, Optional, Union, Awaitable
-{{/asyncio}}
```

Comment:
why these lines are removed?

Reply:
2 reasons:

* These imports are not asyncio-specific
* The imported names are not used in the template

The names could be used by the code generated by the Java code, in which case the imports would be provided from the Java code instead.

Reply:
👌